src/lib.rs

```diff
@@ -299,6 +301,11 @@ impl Check {
         self
     }
 
+    pub fn with_target(&mut self, target: String) -> &mut Self {
```

Comment:
nit: this is a public function so let's document what it does, what's the default value if this function isn't used, etc. It's okay if it mirrors some of the CLI's documentation.